CONTRIBUTING.md

```diff
@@ -0,0 +1,154 @@
+# Contributing to Konflux UI
+
+Thanks for your interest in contributing to **Konflux UI**! 🎉  
+This document will help you get started with contributing code, documentation, or ideas.
+
+---
+
+## How to Contribute
+
+We welcome contributions of all kinds! Follow these steps to get started:
+
+1. **Fork the repository** and create your branch from `main`.
+
+2. Make sure your code builds and passes all checks:
+
+   - Lint (`yarn lint`)
+   - Type check (`yarn tsc`)
+   - Tests (`yarn test`)
```

Comment:
For people who are not in  some orgranition， they do not have permission to run tests.
cc who submitted PR for release monitor dashboard met the permission problem at the beggining.
and i asked why and how we deal with situation, it seems there is no discussion there.

In short, If we mentioned users need to passed tests, we should ensure all users can run tests.